HHS/OPRE-OPS
# adds Additional Information accordion and friends

## What changed

- adds placeholder Additional Information Accordion
- adds placeholder Research-Type-ComboBox
- adds placeholder Special-Topic/Popultaion-Studied-ComboBox
- ComboBox data is not wired up to backend

## Issue

#1870 

## How to test

1. Send Agreement 9 to review
2. Select PLANNED TO EXECUTING
3. See new accordion with ComboBoxes
4. 💰profit

## Screenshots

<img width="1206" alt="SCR-20240117-jnqf" src="https://github.com/HHS/OPRE-OPS/assets/4629398/161fb3c1-d71a-4b0c-80a1-3d34507f4ea0">
<img width="1233" alt="SCR-20240117-jnrk" src="https://github.com/HHS/OPRE-OPS/assets/4629398/7a28c9b2-7932-42fa-9375-1a819fcf6d65">


## Definition of Done Checklist
